docs/en/observability/synthetics-manage-monitors.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[synthetics-manage-monitors]]
+= Manage synthetic monitors
+
+// Purpose of this page: to show users how to maintain their synthetic monitors
+// after they are initially set up.
+
+== Update a synthetic monitor
+
+// Something like... 
+
+To update monitors coming from a test suite you can either:
+
+- Use the push command
+- Update in the integration UI
+- heartbeat?
+
+== Disable a synthetic monitor 
+
+To disable a monitor ... 
```

Comment:
Isn't this somewhat redundant with the section below?

```
[[synthetics-test-suite-send]]
=== Send to {stack}
```

I think it would make sense for this section to only cover the UI, but include a brief explanation about how monitors get to this UI (covering the `push` command in brief).

In other words, I don't think this section should cover `push`, just mention it. Instead, IMO, this section should cover how the UI and the service work.

Reply:
I guess the complication we have is that there is no UI at the moment, other than Monitor Management, which is temporary and hasn't had any documentation (we're not going to, in favour of the new Synthetics UI when that starts to drop in 8.4)

Reply:
>Isn't this somewhat redundant with the section below?

In the existing docs, we don't really explain how to manage monitors _after_ they have already been created (which is what _Send to {stack}_ really covers). That was the intent of this page -- what happens _after_ initial setup.

This would give us an opportunity to explain how `push` is more convenient than managing monitors that were created  via the Elastic Synthetics integration or heartbeat.